backend/core/src/utils/utils.ts

```diff
@@ -0,0 +1 @@
+export const nameof = <T>(name: keyof T) => name
```

Comment:
Is this being used? Maybe add a little docstring about what it does/is for?

Also, while I'm fine having a catch-all "utils" module for stuff that doesn't quite fit in other places, I'm not sure about the `utils/utils.ts` naming - it feels like sub files of `utils` should either be slightly focused/categorized (like `utils/strings.ts`, `utils/types.ts`, etc.) or just `utils/index.ts`.